src/QEQ/fix_qeq_shielded.cpp

```diff
@@ -73,6 +76,22 @@ void FixQEqShielded::init()
 
 }
 
+/* ---------------------------------------------------------------------- */
+
+void FixQEqShielded::extract_reax()
+{
+  Pair *pair = force->pair_match("reax/c",1);
```

Comment:
this fails when using a suffix (e.g. `reax/c/kk`) - may I ask, was that intended (i.e., because it's not supported) or was it an omission?

Reply:
it was probably copied from somewhere. however, since then we can now do smarter matching of styles with some special regular expression characters included and thus i have consolidated the matching to be more flexible, yet specific in PR #1544

Reply:
that was fast :) thanks, works like a charm!